website/content/docs/agent/index.mdx

```diff
@@ -402,3 +459,7 @@ from the load balancer pool.
 The [`skip_leave_on_interrupt`](/docs/agent/options#skip_leave_on_interrupt) and
 [`leave_on_terminate`](/docs/agent/options#leave_on_terminate) configuration
 options allow you to adjust this behavior.
+
+<!-- list of reference-style links -->
+
+[go-sockaddr]: https://godoc.org/github.com/hashicorp/go-sockaddr/template
```

Comment:
```suggestion
[go-sockaddr template]: https://godoc.org/github.com/hashicorp/go-sockaddr/template
```

Reply:
I had pulled this from the Configuration page which has the shorter version.